openedx/core/djangoapps/oauth_dispatch/admin.py

```diff
@@ -5,7 +5,7 @@
 from django.contrib.admin import ModelAdmin, site
 from oauth2_provider import models
 
-from .models import RestrictedApplication
+from .models import RestrictedApplication, OauthRestrictOrganization, OauthRestrictedApplication
```

Comment:
How does `OauthRestrictedApplication` differ from `RestrictedApplication`?  We need a better name.